src/main/features/requirements/views/ca-nextSteps.njk

```diff
@@ -4,10 +4,37 @@
 
 {% set title = 'How to build market event' %}
 
-{% block breadCrumb %}
+{# {% block breadCrumb %}
     {{ CCSBreadcrumbs({
         items: data.breadCrumbs
     }) }}
+{% endblock %} #}
+
+{% block breadCrumb%}
+    {{ CCSBreadcrumbs({
+        items: [
```

Comment:
maybe we should put this info in the json file

Reply:
because of query added to urls, I added it in this way..